packages/o11y-codec-rt/alp/src/lib.rs

```
@@ -111,6 +127,831 @@ pub fn alp_try(val: f64, e: usize) -> Option<i64> {
+        } else {
+            0
+        };
+
+        let exc_bw = if exc_count >= 2 {
+            bits_needed(max_su64 - min_su64) as usize
+        } else {
+            0
+        };
+
+        let exc_full = exc_count * n / sample;
+        let match_bytes = (n * bw + 7) / 8;
```

Clippy (1.95.0)

[warning] manually reimplementing `div_ceil`

(warning)